libs/ngx-dynamic-form-builder/src/lib/utils/dynamic-form-group.ts

```diff
@@ -6,398 +6,135 @@ import { ValidationMetadata } from 'class-validator/metadata/ValidationMetadata'
 import { cloneDeep, mergeWith } from 'lodash-es';
 import 'reflect-metadata';
 import { BehaviorSubject } from 'rxjs';
-export interface IShortValidationErrors {
-  [key: string]: string[] | IShortValidationErrors;
-}
+import { Dictionary, ShortValidationErrors, DynamicFormGroupField } from '../models';
+
 export class DynamicFormGroup<TModel> extends FormGroup {
-  customValidateErrors = new BehaviorSubject<IShortValidationErrors>({});
+
+  public customValidateErrors = new BehaviorSubject<ShortValidationErrors>({});
+
   private _object: TModel;
-  private _externalErrors: IShortValidationErrors;
```

Comment:
Removed the I prefix per Angular guideline recommendations. Can be added back in without issue. https://angular.io/guide/styleguide#interfaces